src/main/java/frc/robot/subsystems/drivetrain/tests/AngleMotorsSpeedTest.java

```diff
@@ -0,0 +1,57 @@
+package frc.robot.subsystems.drivetrain.tests;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+public class AngleMotorsSpeedTest extends CommandBase {
+    private final SwerveDrive swerveDrive;
+    private final Timer timer = new Timer();
+    private final double times[] = new double[4];
+
+    public AngleMotorsSpeedTest(SwerveDrive swerveDrive) {
+        this.swerveDrive = swerveDrive;
+        addRequirements(swerveDrive);
+    }
+
+    @Override
+    public void initialize() {
+        super.initialize();
+        timer.start();
+    }
+
+    @Override
+    public void execute() {
+        swerveDrive.setStates(new SwerveModuleState[]{
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90)),
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90)),
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90)),
+                new SwerveModuleState(0, Rotation2d.fromDegrees(90))
+        });
+        for (int i = 0; i < 4; i++) {
+            if (Math.abs(swerveDrive.getModule(i).getAngle().minus(Rotation2d.fromDegrees(90)).getDegrees()) < 3) {
+                if (times[i] == 0) times[i] = timer.get();
```

Comment:
```suggestion
                if (times[i] == 0)  {
                    times[i] = timer.get();
                }
```
You should always wrap blocks with curly brackets so you won't have problems if you expand the block and forget to add the curly brackets,